woocommerce/woocommerce
# Add "(Experimental)" suffix to experimental blocks

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





In order to decrease confusion, this PR adds "(Experimental)" suffix to the experimental blocks. This is to make it clear they're not publicly available yet.

Changes:
- Add "(Experimental)" suffix to block name
- Add `isExperimentalBlocksEnabled()` usage in `Product Filter: Rating (Experimental)` block
- Regenerate block references - I added a note next to block name if it's unavailable in the inserter
- Update `plugins/woocommerce-blocks/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md` with current experimental blocks and flags
- Add comment reminders to update the docs when updating block registration

Ref: per0F9-1OB-p2



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

To be tested only by devs

1. In `dev` build
2. Go to Editor
3. In the inserter type "Product Filters"
4. **Expected:** There's 1 block with `(Experimental)` suffix:

<img width="353" alt="image" src="https://github.com/woocommerce/woocommerce/assets/20098064/94f2b97a-71f1-49e1-a2d7-48a5d49cf0bc">

5. Insert it
6. Check the tree view
7. Expected: There's `Product Filters (Experimental)` and inside there's 5 `Product Filter (Experimental)` blocks with `Product Filter: SOME_FILTER filter (Experimental)` inside - the key is all of them have `(Experimental)` suffix.

_Note: It's ok the block is truncated. You can hover over three dots next to block to see the full name_

<img width="504" alt="image" src="https://github.com/woocommerce/woocommerce/assets/20098064/fca2b97b-3839-4d2b-b991-c55c5211507d">





### Changelog entry







<details>

#### Significance




#### Type




#### Message 


</details>

<details>

#### Comment 


</details>
